.github/workflows/ci.yml

```
@@ -2,32 +2,82 @@ name: CI
 jobs:
   build:
     runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+      - name: Install node
+        uses: actions/setup-node@v6
+        with:
+          node-version: lts/*
+          cache: npm
+      - name: Install packages (node@${{ steps.node.outputs.node-version }})
```

actionlint (1.7.12)

[error] property "node" is not defined in object type {}

(expression)